src/data/real/ereal.lean

```diff
@@ -514,3 +526,24 @@ lemma to_real_mul : ∀ {x y : ereal}, to_real (x * y) = to_real x * to_real y
 | ⊥ ⊥ := by simp
 
 end ereal
+
+namespace tactic
+
+open positivity
+
+private alias ereal.coe_nonneg ↔ _ ereal_coe_nonneg
+private alias ereal.coe_pos ↔ _ ereal_coe_pos
```

Comment:
This `private` isn't doing anything - you can access the lemma with `tactic.ereal_coe_nonneg` in any later file